content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,151 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: false
+
+title: "EventStorming | Process Modelling"
```

Comment:
I actually went through changing all of these to EventStorming, as this is how Brandolini refers to it on his website: "EventStorming is a flexible workshop format for collaborative exploration of complex business domains." or "What will you use EventStorming for?"

Reply:
It is important that the Practices pages have "one voice" and read as they they were written by the same author. While this is very difficult to do with so many authors, a non-starter is to have two articles about the same topic that spell that topic differently. The [current Event Storming article](https://deploy-preview-943--tanzu-dev-portal.netlify.app/developer/practices/event-storming/) uses the two-word version, and a search across the site shows all current references to the term also use the two-word version https://tanzu.vmware.com/developer/search/?utf8=%E2%9C%93&q=eventstorming#

If we are quoting Brandolini then perhaps using the camel-case version might be appropriate. Again, I recommend chatting with Shaun Andersen as he is leading the practice in this area.

Thoughts?

Reply:
The intended name for the workshop is EventStorming, it may seem mundane but I feel it should be treated like a trade mark, not just a name, as you would for VMware. If we got it wrong then we should correct it. I'm happy to change it elsewhere, I have already updated the event storming practice after a chat with Marc Zottner. I am not confident in getting confirmation from Sean, I already asked for a review and feedback, to which I have had no response. Perhaps @Maarc will be able to confirm what we should do?

Reply:
I guess it is also possible to talk about the exercise as "storming for events" so there may be a middle ground where when talking about the exercise we could say event storming, but when referring to the workshop as a whole, it would be EventStorming by Alberto Brandolini.